nipype/interfaces/base.py

```diff
@@ -1513,13 +1364,12 @@ def _process(drain=0):
             else:
                 for stream in res[0]:
                     stream.read(drain)
+
         while proc.returncode is None:
-            if runtime_profile:
-                mem_mb, num_threads = \
-                    get_max_resources_used(proc.pid, mem_mb, num_threads)
             proc.poll()
             _process()
-            time.sleep(interval)
+            time.sleep(0)
```

Comment:
is `0` what we want here?

Reply:
@oesteban - i made this comment here - that's how long it has been since i wrote that code. now i think it may be there to create a no-op. 

but it shouldn't be `poll_sleep_duration` since this is trying to get stdout/error as quickly as possible and report on the screen.

Reply:
0 was my suggestion to just yield control rather than busy wait, but 0.1 is probably fast enough.

Reply:
It wasn't there before the ResourceMultiProc was merged, so I wonder if it is really necessary.

Reply:
perhaps we can try without it and see what happens.